### PR TITLE
punktf-cli: Add note when executing dry-run

### PR DESCRIPTION
**Pull request by michidk**
Not sure whether this is the right location for that part

### CODE REVIEW

File: crates/punktf-cli/main.rs [hunk 1]
```diff
@@ -251,6 +251,12 @@ fn handle_commands(opts: opt::Opts) -> Result<()> {
 					if deployment.status().is_failed() {
 						Err(eyre!("Some dotfiles failed to deploy"))
 					} else {
+						if options.dry_run {
+							log::info!(
+								"Note: No files were actually deployed, since dry run mode was \
+								 eabled"
```
Comment (Shemnei) [inline crates/punktf-cli/main.rs:257]: Should be 'enabled'

File: crates/punktf-cli/main.rs [hunk 2]
```diff
@@ -251,6 +251,12 @@ fn handle_commands(opts: opt::Opts) -> Result<()> {
 					if deployment.status().is_failed() {
 						Err(eyre!("Some dotfiles failed to deploy"))
 					} else {
+						if options.dry_run {
```
Comment (Shemnei) [inline crates/punktf-cli/main.rs:254]: I would put this check before the failed check on line 251. That way this info is also printed if an error occured.

**Review by Shemnei — CHANGES_REQUESTED** [review]

File: crates/punktf-cli/main.rs (hunk 1 above)
Comment (michidk) [reply crates/punktf-cli/main.rs:257]: dang it. thx!

File: crates/punktf-cli/main.rs (hunk 2 above)
Comment (michidk) [reply crates/punktf-cli/main.rs:254]: True.

**Review by Shemnei — APPROVED** [review]
Nice

**Pull request merged**